README.md

```diff
@@ -1 +1,11 @@
-# course-selection
\ No newline at end of file
+# course-selection
+# Running the Program -- Frontend.
+To run the program, cd into the front_end directory.
+Run ```npm install``` to install the required packages.
+Use ```npm start``` to run the program. It will run on localhost:3000
+
+
+# Running the Program -- Backend.
+To run the program, ensure that your .env files are set up. See back_end README.md for instructions.
+
+Then, use ```flask run``` to run the program. It will run on localhost:5000
```

Comment:
Thanks for documenting this! We should probably expand this readme even further at some point with more specific instructions, as well as an overview of the project (in a future pull request, of course, not here!).